__test__/integration/debt_token_api/runners/index.ts

```diff
@@ -76,15 +73,19 @@ export abstract class ScenarioRunner {
         return order;
     }
 
-    private async getDummyTokenBySymbol(
-        web3: Web3,
-        testAPIs: TestAPIs,
-        tokenSymbol: string,
-    ): Promise<DummyTokenContract> {
-        const { contractsAPI } = testAPIs;
+    public tokenIDForOrder = async (loanOrder: SimpleInterestLoanOrder): Promise<BigNumber> => {
```

Comment:
Could we choose a naming scheme that more explicitly indicates the fact that the order is being filled by this method?

I feel like my first intuition reading this was that this imply takes an order and returns its corresponding token ID

Reply:
Agreed